utils/to-doc.js

```diff
@@ -1,12 +1,16 @@
 'use strict'
 
 var extend = require('pouchdb-extend')
+var now = require('./now')
 
 module.exports = function objectToDoc (object) {
   var doc = extend({}, object, {
     _id: object.id
   })
 
+  doc.updatedAt = now()
+  doc.createdAt = doc.createdAt || doc.updatedAt
```

Comment:
Trouble is that is used after we've got a response but we need to store the timestamps, so maybe let's go with the dedicated method.